integration/testrunner/grpc.go

```diff
@@ -69,6 +72,12 @@ var (
 	privateKey, _    = x509.ParseECPrivateKey(privateKeyPEM.Bytes)
 )
 
+type GRPCFields struct {
```

Comment:
I don't think this is what we want. We should compare log entry instead. Any reason why we can't do that?

Reply:
+1

Reply:
I checked the fields of our LogEntry emitted by lumberjack, I still feel it is doable to compare log entry instead